src/toil/test/src/jobCacheTest.py

```diff
@@ -0,0 +1,848 @@
+# Copyright (C) 2015 UCSC Computational Genomics Lab
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Author : Arjun Arkal Rao
```

Comment:
Not a big fan of vanity headers for open source software. Toil is a team effort and many people will touch this code. You will be listed as the commit author. If you want, you can start a ACKNOWLEDGEMENTS file in the project root and put your name there.